code/web/src/modules/survey/SurveyPage.js

```diff
@@ -3,43 +3,203 @@ import React, { PureComponent } from 'react'
+								</div>
+								<input type="radio" style={{ marginLeft: '50%' }} />
+							</Card>
+						</GridCell>
+					</Grid>) : null
+				}
+				<Grid>
+					<H4 font="secondary" style={{ marginLeft: '.2em', marginTop: '1em' }}>Shoes</H4>
+				</Grid>
+				{this.state.parsedItems.rocker.length > 0 ? 
+					(<Grid style={{ height: '24vh', width: '100vw', display: 'flex', flexWrap: 'nowrap', overflowX: 'auto' }}>
+						<GridCell>
+							<Card style={{ marginTop: '2em', width: '19em', backgroundColor: white, marginLeft: '9px' }} key='rocker-shoes'>
+								<div style={{ display: 'flex' }}>
+									<img src={routeImage + this.state.parsedItems.rocker[5].image} alt={this.state.parsedItems.rocker[5].name} style={{ width: '50%' }}/>
+									<img src={routeImage + this.state.parsedItems.rocker[6].image} alt={this.state.parsedItems.rocker[6].name} style={{ width: '50%' }}/>
+								</div>
+								<input type="radio" style={{ marginLeft: '50%' }} />
+							</Card>
+						</GridCell>
+						<GridCell>
+							<Card style={{ marginTop: '2em', width: '19em', backgroundColor: white, marginLeft: '9px' }} key='bohemian-shoes'>
+								<div style={{ display: 'flex' }}>
+									<img src={routeImage + this.state.parsedItems.bohemian[5].image} alt={this.state.parsedItems.bohemian[5].name} style={{ width: '50%' }}/>
+									<img src={routeImage + this.state.parsedItems.bohemian[6].image} alt={this.state.parsedItems.bohemian[6].name} style={{ width: '50%' }}/>
+								</div>
+								<input type="radio" style={{ marginLeft: '50%' }} />
+							</Card>
+						</GridCell>
+						<GridCell>
+							<Card style={{ marginTop: '2em', width: '19em', backgroundColor: white, marginLeft: '9px' }} key='business-shoes'>
+								<div style={{ display: 'flex' }}>
+									<img src={routeImage + this.state.parsedItems.business[5].image} alt={this.state.parsedItems.business[5].name} style={{ width: '50%' }}/>
+									<img src={routeImage + this.state.parsedItems.business[6].image} alt={this.state.parsedItems.business[6].name} style={{ width: '50%' }}/>
+								</div>
+								<input type="radio" style={{ marginLeft: '50%' }} />
+							</Card>
+						</GridCell>
+						<GridCell>
+							<Card style={{ marginTop: '2em', width: '19em', backgroundColor: white, marginLeft: '9px' }} key='artsy-shoes'>
+								<div style={{ display: 'flex' }}>
+									<img src={routeImage + this.state.parsedItems.artsy[5].image} alt={this.state.parsedItems.artsy[5].name} style={{ width: '50%' }}/>
+									<img src={routeImage + this.state.parsedItems.artsy[6].image} alt={this.state.parsedItems.artsy[6].name} style={{ width: '50%' }}/>
+								</div>
+								<input type="radio" style={{ marginLeft: '50%' }} />
+							</Card>
+						</GridCell>
+					</Grid>) : null
+				}
+
+				<Grid>
+				  <Button theme="secondary" style={{ marginLeft: '50%', marginTop: '2em', marginBottom: '2em' }}>Submit</Button>
+				</Grid>
```

Comment:
A bit hard coded to allow for inclusive design. Ideally with more time this could be more elegant, with an iterator method.